lib/pickle.js

```diff
@@ -107,6 +107,9 @@ var PickleIterator = (function () {
 // space is controlled by the header_size parameter passed to the Pickle
 // constructor.
 var Pickle = (function () {
+  var emptyBuffer = typeof Buffer.alloc === 'function'
+    ? Buffer.alloc(0) : new Buffer(0)
```

Comment:
Am I reading this right that every Pickle will now be initialized to point to the _same_ empty buffer? This seems inherently unsafe